### ISSUE TITLE
Выделение оригинальных предложений из текста "как есть"

### ISSUE THREAD

**Issue by mnvx** [yandex/tomita-parser#50]
При выделении предложений из текста с помощью Томита-парсера не удается справиться со следующей особенностью.

Текст для анализа:

> Мама мыла раму, т.к. папа был занят.

Текст, который получаю в результате разбора в html (`PrettyOutput`):

> Мама   мыла   раму   ,   т.к   .   папа   был   занят   .   EOS 

Аналогично в текстовом файле:

> Мама мыла раму , т.к . папа был занят .

В этом тексте добавлены пробелы после слова "мама", "т.к." и "занят". Возможно ли в выводе получить текст "как есть", без добавленных пробелов?

Мой конфиг:

```
encoding "utf8";

TTextMinerConfig {
  Dictionary = "mydic.gzt";

  PrettyOutput = "debug.html";

  Input = {
    File = "test.txt";
  }

  Output = {
    Format = text;
    File = "out.txt";
  }
}
```

mydic.gzt:
```
encoding "utf8";               // указываем кодировку, в которой написан этот файл

import "base.proto";           // подключаем описания protobuf-типов (TAuxDicArticle и прочих)
import "articles_base.proto";  // Файлы base.proto и articles_base.proto встроены в компилятор.
                               // Их необходимо включать в начало любого gzt-словаря.
```

**Comment by victorbocharov** [comment]
> Возможно ли в выводе получить текст "как есть", без добавленных пробелов?

Нет. Пробелы отражают работу токенизатора. Это отладочная информация для разработчика, чтобы было проще понять, почему выделились те или иные факты.
